pkg/spec/spec.go

```diff
@@ -32,7 +32,8 @@ type Function struct {
 // FunctionSpec contains func specification
 type FunctionSpec struct {
 	Handler  string             `json:"handler"`
-	Function string             `json:"function"`
+	File     string             `json:"file"`
```

Comment:
I greatly dislike losing the ability to provide the function text inline.  Is there some way we can make this a URL (and support "data:" URLs), or continue to accept the old `Function` text in addition to a list of `Files` (or something like that).